### PR TITLE
Review redirect.pt.Rmd

### PR DESCRIPTION
**Pull request by IldySilva**
the #777 
@yabellini ,I would like to contribute to the project, please add me to the Slack channel,

### CODE REVIEW

**Comment by yabellini** [comment]
This is great @IldySilva !  Thank you :-) Can you email me at yabellini@ropensci.org so I can send the Slack invitation?

File: redirect.pt.Rmd [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Como definir um redirecionamento {#redirect}
 
-## Site que não seja de páginas do GitHub (por exemplo, Netlify) {#non-git-hub-pages-site-e-g-netlify}
+## Site que não seja de páginas do Github Pages (por exemplo, Netlify) {#non-git-hub-pages-site-e-g-netlify}
 
-Substituir o conteúdo do site atual por um site `index.html` e `404.html` contendo ambos os arquivos:
+Substitua o conteúdo do site atual por um arquivos com o nome `index.html` e um com nome  `404.html` contendo ambos os arquivos:
```
Comment (pedropark99) [inline redirect.pt.Rmd:5]:
```suggestion
Substitua o conteúdo do site atual por dois arquivos chamados `index.html` e `404.html`. Ambos os arquivos devem conter o seguinte conteúdo:
```

Comment (pedropark99) [reply redirect.pt.Rmd:5]: This makes the intent of the phrase more clear

**Review by pedropark99 — CHANGES_REQUESTED** [review]

**Comment by pedropark99** [comment]
@yabellini  this PR is ready to be merged since both reviews are done 😉

**Pull request merged**